### DISCUSSION TITLE
零氪绳匠想问一下2命格莉斯怎么练

### DISCUSSION THREAD

**Discussion by justsmile3** [share121/inter-knot#2605] [其他]
![image](https://github.com/user-attachments/assets/e2d28c1a-7466-4314-b777-799b6da505d1)

**Comment by qmtqyh** [comment]
同问呜呜呜

**Comment by Kylez416** [comment]
看看下半能不能和简，塞斯组个队感觉有的搞

**Comment by Nie7bai** [comment]
可以带四雷套直接打感电；
如果有派派想玩紊乱的话带那个精通四件套；
精精掌或者精精精；
不是很精通，个人理解，谨慎采纳<img src="https://img-static.mihoyo.com/communityweb/upload/408018316100a8a8a846c68acedaa191.gif" alt="米游姬 卖萌" width="75" >

**Reply by Onefeilinmiao** [reply]
要么纯电四雷，队友露西，丽娜；要么紊乱四蓝调，队友派派/妮可，加其他拐，看你盒子有哪些了

**Comment by justsmile3** [comment]
![1723996564611](https://github.com/user-attachments/assets/1f46cad9-0f3b-4787-9f76-e8fc538ca9b8)
我的角色

**Comment by justsmile3** [comment]
我现在不知道练谁了

**Comment by Onefeilinmiao** [comment]
个人见解，你这个盒子不太适合玩格莉丝，硬要玩就和派派组队吧，派派1命以上能玩，再加个妮可或者安东，另一队就是苍角主C，带个青衣和其他辅助。不过还是推荐屯点票抽个1.2的新角色火异常很配格莉丝，新角色是后台，格莉丝站场打

**Reply by Onefeilinmiao** [reply]
是安比不是安东，不好意思打错了

**Reply by justsmile3** [reply]
谢谢，我主要是看着她2命了不用有点浪费哈哈哈，就和我星铁一样分币没氪白露专武精四了

**Reply by Onefeilinmiao** [reply]
没事呀，格莉丝算常驻里很厉害的了，配丽娜和露西的话，不过丽娜最好得有1命，露西2命，然后配格莉丝和丽娜专武，格莉丝4雷2穿透，5号位带穿透。等1.2火异常一出就可以把露西替换掉

**Reply by justsmile3** [reply]
仓角还能打主c流 啊

**Reply by Onefeilinmiao** [reply]
可以，但仅限于满命且最好是有狼叔，但没有狼叔苍角的伤害也比你的猫又伤害高
